### PR TITLE
fix: remove trailing commas from keystore json

### PR DESCRIPTION
**Pull request by weboko**
# Description
In this PR I remove trailing commas from Keystore `json`. 

## How to test
Tests essentially do not change as it is not a breaking change from `nwaku` standpoint.

## Issue

closes https://github.com/waku-org/js-rln/issues/80

### CODE REVIEW

File: waku/waku_keystore/keyfile.nim
Comment (weboko) [inline waku/waku_keystore/keyfile.nim:376]: dunno if it the best way to address it 🤷‍♂️

**Comment by alrevuelta** [comment]
Can @rymnc have a look?

**Review by rymnc — COMMENTED** [review]
will look at an alternative in some time!

**Comment by weboko** [comment]
@rymnc what are alternatives you are looking at?

**Review by rymnc — APPROVED** [review]
LGTM, if you can get checks to pass!

**Comment by weboko** [comment]
@alrevuelta any comments from you?

**Comment by alrevuelta** [comment]
> @alrevuelta any comments from you?

No strong opinion as long as it's backward compatible. Delegating on @rymnc since its the code owner.

> In this PR I remove trailing commas from Keystore json.

Might be worth adding why doing so. Not very clear to me.

**Comment by weboko** [comment]
Here is explanation ser - https://github.com/waku-org/js-rln/issues/80#issuecomment-1802326819
If all good from your side - please, approve

**Comment by alrevuelta** [comment]
> Here is explanation ser - https://github.com/waku-org/js-rln/issues/80#issuecomment-1802326819
So it seems that

> Keystore generated by nwaku is not a valid JSON.

Is that the case @rymnc. Will be break backward compatibility with old keystores?

**Review by alrevuelta — APPROVED** [review]
Approving, ensure CI passes and existing keystores are compatible with this new changes.

**Comment by Ivansete-status** [comment]
Ah, unfortunately, the tests are failing:
image

File: waku/waku_keystore/keyfile.nim
Comment (Ivansete-status) [inline waku/waku_keystore/keyfile.nim:345]: Nitpick and not urgent but I'd either put it at the top of the file or in another `keyfile_types.nim` file.

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks for it :)!
Just added a minor nitpick comment

**Pull request merged**
